pinot-tools/src/main/java/org/apache/pinot/tools/data/generator/DataGenerator.java

```diff
@@ -100,6 +110,23 @@ public void generate(long totalDocs, int numFiles)
     }
   }
 
+  public void generateCsv(long totalDocs, int numFiles)
+      throws IOException {
+    final int numPerFiles = (int) (totalDocs / numFiles);
+    for (int i = 0; i < numFiles; i++) {
+      try (FileWriter writer = new FileWriter(outDir + "/output.csv")) {
```

Comment:
Output file name should come from cli?

Reply:
cleaned up

Reply:
output is directory-based due to avro support above. imho doesn't add much value to support custom file names

Reply:
Won't this always override the previous file?

Reply:
the `outDir` is set as a cli arg

Reply:
btw. here's the `generator.sh` script that uses the DataGenerator:
https://github.com/apache/incubator-pinot/blob/master/docker/images/pinot/bin/generator.sh